timemachine/ff/__init__.py

```diff
@@ -69,6 +73,8 @@ class Forcefield:
         ]
     ]
     lj_handle: Optional[nonbonded.LennardJonesHandler]
+    lj_handle_solv: Optional[nonbonded.LennardJonesSolventHandler]
+    lj_handle_intra: Optional[nonbonded.LennardJonesIntraHandler]
```

Comment:
nit: is there a canonical order for `lj_handle, lj_handle_solv, lj_handle_intra`?

`parameterize_nonbonded` uses a different order `lj_handle, lj_handle_intra, lj_handle_solv`, which seems error-prone.